routes/form_action.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const utils = require('../helpers/utils');
+const router = express.Router();
+
+async function action(page, request) {
+    const inputMapping = request.body.inputMapping;
+    const submitButton = request.body.submitButton || null;
+
+    for (const [selector, params] of Object.entries(inputMapping)) {
+        const value = params.value || "no value was provided";
+        const delay = params.delay || 0;
+        await page.type(selector, value, { delay });
+    }
+
+    if (submitButton) {
+        await page.click(submitButton);
+    }
+
+    return await utils.getContents(page);
+
+}
+
+// body = {
+//     "inputMapping": { A dictionary where each key is a CSS selector, and each value is another dictionary containing details about the input for that element:
+//         "selector": <string> The CSS selector for the input element (used as the key).
+//         "value": <string> The text to be inputted into the element.
+//         "delay": <number> A delay (in milliseconds) between each keystroke when inputting the text. Defaults to 0 if not provided.
+//     },
+//     "submitButton": <string> The CSS selector for the form's submit button. If provided, the button will be clicked after filling in the form.
+// }
+//
+router.post('/', async function (req, res, next) {
+
```

Comment:
May you should check absence of `submitButton` and `inputMapping` as we do in other route modules.

Reply:
Made the check for inputMapping.
submitButton is optional.